public/app/features/explore/LogsContainer.tsx

```diff
@@ -133,7 +133,7 @@ export class LogsContainer extends PureComponent<PropsFromRedux & LogsContainerP
               width={width}
               getRowContext={this.getLogRowContext}
               getFieldLinks={this.getFieldLinks}
-              queries={queries}
+              logsQueries={logsQueries || []}
```

Comment:
```suggestion
              logsQueries={logsQueries ?? []}
```

i like `??` 😁  (same argument as the one above (untested change)

Reply:
This was changed to `logsQueries={logsQueries}` as logsQueries is now optional param.